src/sections/collection/collection-items-panel/search-panel/SearchPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Button, Form } from '@iqss/dataverse-design-system'
+import { Search } from 'react-bootstrap-icons'
+import styles from './SearchPanel.module.scss'
+
+interface SearchPanelProps {
+  currentSearchValue?: string
+  isLoadingCollectionItems: boolean
+  onSubmitSearch: (searchValue: string) => void
+}
+
+export const SearchPanel = ({
+  currentSearchValue = '',
+  isLoadingCollectionItems,
+  onSubmitSearch
+}: SearchPanelProps) => {
+  const { t } = useTranslation('collection')
+
+  const [searchValue, setSearchValue] = useState(currentSearchValue)
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    const trimmedSearchValue = searchValue.trim()
+
+    const encodedSearchValue = encodeURIComponent(trimmedSearchValue)
+
+    onSubmitSearch(encodedSearchValue)
+  }
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(event.target.value)
+  }
+
+  useEffect(() => {
+    setSearchValue(currentSearchValue)
+  }, [currentSearchValue])
+
+  return (
+    <div className={styles['search-panel']}>
+      <form onSubmit={handleSubmit} className={styles['search-form']} role="search">
+        <Form.InputGroup className={styles['search-input-group']}>
+          <Form.Group.Input
+            type="search"
+            placeholder={t('searchThisCollectionPlaceholder')}
+            aria-label="Search"
+            value={searchValue}
+            onChange={handleSearchChange}
+          />
+          <Button
+            variant="secondary"
+            type="submit"
+            icon={<Search />}
+            aria-label={t('searchSubmitButtonLabel')}
+            disabled={isLoadingCollectionItems}
+          />
+        </Form.InputGroup>
+      </form>
+      {/* <Button variant="link" className={styles['advanced-search-btn']} disabled>
```

Comment:
is this comment needed?

Reply:
It is the button to use when we do the advanced search, it already has the styles applied and the responsive design is also done. I think it would be good to leave it commented for now for when the time comes to do the advanced search, do you think?

Reply:
ok, makes sense!  Maybe just add a comment explaining that it's for the future